Makefile

```diff
@@ -120,20 +123,25 @@ build-java-no-tests:
 # Go SDK
 
 install-go-ci-dependencies:
-	go get -u github.com/golang/protobuf/protoc-gen-go
-	go get -u golang.org/x/lint/golint
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26.0
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
+
+compile-protos-go: install-go-ci-dependencies
+	pip install -e "sdk/python[ci]"
```

Comment:
Oh it's super dumb. We rely on `grpc_tools.protoc` to compile things for go as well because we do it using `python setup.py build_go_protos`. But `grpc_tools.protoc` comes from `grpcio-tools` which has to be installed by installing something from `setup.py`. 

Ideally.. we have a target to do a minimal install of the libraries needed to build feast itself.

Reply:
One of my todos is to redo the Makefile so I'll leave it as is